website/docs/r/vpc_ipv6_cidr_block_association.html.markdown

```diff
@@ -10,8 +10,8 @@ description: |-
 
 Provides a resource to associate additional IPv6 CIDR blocks with a VPC.
 
-When a VPC is created, a primary IPv6 CIDR block for the VPC must be specified.
-The `aws_vpc_ipv6_cidr_block_association` resource allows further IPv6 CIDR blocks to be added to the VPC.
+When a VPC is created, a primary IPv4 CIDR block for the VPC must be specified.
+The `aws_vpc_ipv6_cidr_block_association` resource an IPv6 CIDR block to be added to the VPC.
```

Comment:
```suggestion
The `aws_vpc_ipv6_cidr_block_association` resource allows IPv6 CIDR blocks to be added to the VPC.
```

Reply:
The IPv4 resource uses the plural form despite each association only adding a single block at a time. But you can use multiple of them to add multiple blocks, so the plural form might be better.